src/components/WebexMeetingInfo/WebexMeetingInfo.test.js

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+
+import WebexMeetingInfo, {formatMeetingTime} from './WebexMeetingInfo';
+
+jest.mock('../hooks/useMeeting');
+
+describe('Webex Meeting Info component', () => {
+  describe('snapshots', () => {
+    test('matches snapshot of scheduled meeting', () => {
+      expect(shallow(<WebexMeetingInfo meetingID="scheduledMeeting" />)).toMatchSnapshot();
+    });
+
+    test('matches snapshot of one on one meeting', () => {
+      expect(shallow(<WebexMeetingInfo meetingID="oneOnOneMeeting" />)).toMatchSnapshot();
+    });
+
+    test('matches snapshot of space meeting', () => {
+      expect(shallow(<WebexMeetingInfo meetingID="spaceMeeting" />)).toMatchSnapshot();
+    });
+
+    test('matches snapshot of no meeting title', () => {
+      expect(shallow(<WebexMeetingInfo meetingID="noMeetingTitle" />)).toMatchSnapshot();
+    });
+
+    test('matches snapshot of loading', () => {
+      expect(shallow(<WebexMeetingInfo meetingID="loadingMeeting" />)).toMatchSnapshot();
+    });
+  });
+
+  describe('formatMeetingTime() returns', () => {
+    test('a formatted start and end time string', () => {
+      const startTime = new Date('2019-08-20T21:00:00.000Z');
+      const endTime = new Date('2019-08-20T22:00:00.000Z');
+
+      expect(formatMeetingTime(startTime, endTime)).toMatchSnapshot();
```

Comment:
I just realized this is a snapshot, should this be an regular assert? e.g. `toEqual`?
```suggestion
      expect(formatMeetingTime(startTime, endTime)).toEqual('5:00 PM - 6:00 PM');
```